### PR TITLE
F#4 moveitlab connection

### PR DESCRIPTION
**Pull request by dg-shadow**

### CODE REVIEW

**Comment by ugocupcic** [comment]
Cool! Don't forget the other moveit config (left arm / hand only)

**Comment by dg-shadow** [comment]
I've moved the remote connection stuff into its own package.

One thing - the hand alone doesn't have it's own moveit config as far as I
can tell, so doesn't have the same logical place to include the remote
launch stuff. As it stands you just have to launch sr_remote_warehouse
remote_warehouse.launch manually. Thoughts?

On 10 September 2015 at 05:39, Ugo Cupcic notifications@github.com wrote:

> Cool! Don't forget the other moveit config (left arm / hand only)
> 
> —
> Reply to this email directly or view it on GitHub
> https://github.com/shadow-robot/sr_interface/pull/11#issuecomment-139114037
> .

## 

_Dan Greenwald_
Design Engineer

+44 (0) 20 3051 8149 ext 316
dg@shadowrobot.com
www.shadowrobot.com

## 

---

_Shadow Robot Company Ltd._
251 Liverpool Road, N1 1LX, UK
Registered Number 3308007 (England & Wales)

RoNeX - Building Robots with ROS Made Easy 

## http://www.shadowrobot.com/products/ronex/

File: sr_remote_warehouse/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,188 @@
+cmake_minimum_required(VERSION 2.8.3)
```
Comment (ugocupcic) [inline sr_remote_warehouse/CMakeLists.txt]: That's a very good idea, but put it in sr_config since it is something that could be modified by the end user.

**Comment by ugocupcic** [comment]
I forgot to [put it](https://github.com/shadow-robot/sr-contrib/tree/indigo-devel/sr_moveit_hand_config) in here. I'll send a pr. My bad.

**Comment by ugocupcic** [comment]
Here's the pr for the moveit hand config https://github.com/shadow-robot/sr_interface/pull/12

**Comment by ugocupcic** [comment]
@dg-shadow the pr has been merged so you can add the remote db to this as well (the remote db supports multiple robots right?)

File: sr_multi_moveit/right_sr_ur10_moveit_config/launch/remote_warehouse.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+warehouse_authenticate: true
```
Comment (ugocupcic) [inline sr_multi_moveit/right_sr_ur10_moveit_config/launch/remote_warehouse.yaml]: shouldn't that be deleted now?

File: sr_multi_moveit/right_sr_ur10_moveit_config/launch/warehouse_settings.launch.xml [hunk 3]
```diff
@@ -7,9 +7,9 @@
   <!-- The default DB host for moveit -->
   <arg name="moveit_warehouse_host" default="localhost"/>
 
+
   <!-- Set parameters for the warehouse -->
   <param name="warehouse_port" value="$(arg moveit_warehouse_port)"/>
   <param name="warehouse_host" value="$(arg moveit_warehouse_host)"/>
-  <param name="warehouse_exec" value="mongod"/>
```
Comment (ugocupcic) [inline sr_multi_moveit/right_sr_ur10_moveit_config/launch/warehouse_settings.launch.xml:13]: not needed?

File: sr_moveit_hand_config/launch/default_warehouse_db.launch [hunk 4]
```diff
@@ -2,12 +2,21 @@
 
   <arg name="reset" default="false"/>
```
Comment (ugocupcic) [inline sr_moveit_hand_config/launch/default_warehouse_db.launch:5]: missing `<arg name="remote_warehouse" default="false"/>`

**Pull request merged**